data_juicer/ops/base_op.py

```diff
@@ -133,6 +133,7 @@ def __init__(self, *args, **kwargs):
         self.image_key = kwargs.get('image_key', 'images')
         self.audio_key = kwargs.get('audio_key', 'audios')
         self.video_key = kwargs.get('video_key', 'videos')
+        self.batch_size = kwargs.get('batch_size', 1)
```

Comment:
Should we change the default value to 1000 to align with the default values of HF `dataset.map/filter`?

Reply:
Then 1000 might be better to align with users' expectations of existing tools.